app/models/portfolio.rb

```diff
@@ -2,6 +2,7 @@ class Portfolio < ApplicationRecord
   include OwnerField
   include Discard::Model
   include Catalog::DiscardRestore
+  include Aceable
```

Comment:
Should we add this to PortfolioItem as well since those currently aren't behind RBAC? I know we had a talk scheduled about it at the all hands, right @gmcculloug ?

Reply:
I would suggest we address filtering PortfolioItems in a different PR so we do not  overcomplicate this one.  And I think PortfolioItems will be handled differently as access will not be directly applied to that object but based on the access to the parent Portfolio.